.github/workflows/build-test-prValidation.yml

```diff
@@ -89,16 +87,15 @@ jobs:
           # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
           fetch-depth: 0
 
-      - name: Set up JDK 8
+      - name: Set up JDK 11
         uses: olafurpg/setup-scala@v10
         with:
-          java-version: adopt@1.8.0
+          java-version: adopt@1.11
```

Comment:
didn't we use 8 for pr validation in Jenkins? it can be pretty good to catch such mistakes early in validation, since we support 8 but all of us use 11 for development